sdk/ml/azure-ai-ml/setup.py

```diff
@@ -80,15 +80,14 @@
         "pyjwt<=2.3.0",
         "azure-storage-blob<=12.9.0,>=12.5.0",
```

Comment:
we need to update this pinning as it's causing a build failure since azure-storage-file-datalake has a different dependency range.

Reply:
resolved